resources/markdown/samReadme.md

```diff
@@ -0,0 +1,32 @@
+# Developing AWS SAM Applications with the ${PRODUCTNAME}
+
+This project contains source code and supporting files for a serverless application that you can locally run, debug, and deploy to AWS with the ${PRODUCTNAME}.
```

Comment:
We need to be as un-verbose as possible.
```suggestion
This project contains source code for a serverless ("SAM") application that you can debug locally and deploy to AWS.
```

Reply:
This line is the one and only direct rip I took from the existing readme 🙃 

https://github.com/aws/aws-sam-cli-app-templates/blob/master/nodejs14.x/cookiecutter-aws-sam-hello-nodejs/%7B%7Bcookiecutter.project_name%7D%7D/README.md#-cookiecutterproject_name-

Reply:
yeah. We will keep narrowing and refining the text here over time. We have that luxury, because we only have 1 file to focus on :)